File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocso/OCS2016U00LoadDiscussionInfo.java
package nta.med.data.model.ihis.ocso;

public class OCS2016U00LoadDiscussionInfo {

	private String doctor;
	private String doctorName;
	private String content;
	private String updated;
	private String editedFlg;
	private String grpQuestionId;
	private String discussionId;

	public OCS2016U00LoadDiscussionInfo(String doctor, String doctorName, String content, String updated,
			String editedFlg, String grpQuestionId, String discussionId) {
		super();
		this.doctor = doctor;
		this.doctorName = doctorName;
		this.content = content;
		this.updated = updated;
		this.editedFlg = editedFlg;
		this.grpQuestionId = grpQuestionId;
		this.discussionId = discussionId;
	}

	public String getDoctor() {
		return doctor;
	}

	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}

	public String getDoctorName() {
		return doctorName;
	}

	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}

	public String getContent() {
		return content;
	}

	public void setContent(String content) {
		this.content = content;
	}

	public String getUpdated() {
		return updated;
	}

	public void setUpdated(String updated) {
		this.updated = updated;
	}

	public String getEditedFlg() {
		return editedFlg;
	}

	public void setEditedFlg(String editedFlg) {
		this.editedFlg = editedFlg;
	}

	public String getGrpQuestionId() {
		return grpQuestionId;
	}

	public void setGrpQuestionId(String grpQuestionId) {
		this.grpQuestionId = grpQuestionId;
	}

	public String getDiscussionId() {
		return discussionId;
	}

	public void setDiscussionId(String discussionId) {
		this.discussionId = discussionId;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocso/OCS2016U00LoadDiscussionInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocso
{
    [Serializable]
    public class OCS2016U00LoadDiscussionInfo
    {
        private String _doctor;
        private String _doctorName;
        private String _content;
        private String _updated;
        private String _editedFlg;
        private String _grpQuestionId;
        private String _discussionId;
        private String _dataRowState;

        public String Doctor
        {
            get { return this._doctor; }
            set { this._doctor = value; }
        }

        public String DoctorName
        {
            get { return this._doctorName; }
            set { this._doctorName = value; }
        }

        public String Content
        {
            get { return this._content; }
            set { this._content = value; }
        }

        public String Updated
        {
            get { return this._updated; }
            set { this._updated = value; }
        }

        public String EditedFlg
        {
            get { return this._editedFlg; }
            set { this._editedFlg = value; }
        }

        public String GrpQuestionId
        {
            get { return this._grpQuestionId; }
            set { this._grpQuestionId = value; }
        }

        public String DiscussionId
        {
            get { return this._discussionId; }
            set { this._discussionId = value; }
        }

        public String DataRowState
        {
            get { return this._dataRowState; }
            set { this._dataRowState = value; }
        }

        public OCS2016U00LoadDiscussionInfo() { }

        public OCS2016U00LoadDiscussionInfo(String doctor, String doctorName, String content, String updated, String editedFlg, String grpQuestionId, String discussionId, String dataRowState)
        {
            this._doctor = doctor;
            this._doctorName = doctorName;
            this._content = content;
            this._updated = updated;
            this._editedFlg = editedFlg;
            this._grpQuestionId = grpQuestionId;
            this._discussionId = discussionId;
            this._dataRowState = dataRowState;
        }

    }
}